notebook/Solution/04 - LSTM_Multivariate_Univariate_SingleStep.ipynb

```diff
@@ -330,28 +330,79 @@
     "print(test_data.shape)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "21"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "len(train_data['in_seq1'])"
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "## Data Standardization\n",
-    "Standardize the data to have 0 mean and unit variance in order to make our model converge faster."
+    "Standardize the data based on every column to have 0 mean and unit variance to make our model converge faster. "
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 9,
    "metadata": {},
    "outputs": [],
    "source": [
-    "scaler = StandardScaler().fit(train_data.values.reshape(-1,1))\n",
-    "train_data_standard = scaler.transform(train_data.values.reshape(-1, 1))\n",
-    "test_data_standard = scaler.transform(test_data.values.reshape(-1, 1))"
+    "def multi_data_scaler(train_data,test_data,scale_mode = \"Standardize\"):   \n",
```

Comment:
Suppose this is inserted intentionally and not by accident? bcuz other nbs just call from your library. If it's an intentional inclusion, ignore my cmt, else pls remove it thanks

Reply:
yes, this function is created to solve the bug of performing data scaling in one shot. The correct way is to scale the data column by column. 
This function only appears in 04 - LSTM_Multivariate_Univariate_SingleStep.ipynb because this notebook will be the first multivariate data for the student to hand on.

Reply:
pycache is already in .gitignore, so not action from me 
![image](https://user-images.githubusercontent.com/59526258/119109688-f6c3a180-ba53-11eb-94a1-f652f7c85f06.png)

Reply:
Thanks for the reviewing @KianYang-Lee

Reply:
Please can you take some actions because it is shown here that it is tracked by Git. Best way would be to delete `pycache`. Thank you
![image](https://user-images.githubusercontent.com/49669630/120142074-39ce0380-c210-11eb-817e-81a1f7f57d2e.png)

Reply:
Deleted the __pycache__ folder. Guess it couldn't resist from tracking the .pyc files within that folder because it doesn't recognise the folders that start with underscores as a directory.